pallets/parachain-system/src/lib.rs

```diff
@@ -1067,6 +1067,33 @@ impl<T: Config> Pallet<T> {
 			)],
 		})
 	}
+
+	/// Prepare/insert relevant data for `schedule_code_upgrade` for benchmarks.
+	#[cfg(feature = "runtime-benchmarks")]
+	pub fn initialize_for_set_code_benchmark(max_code_size: u32) {
+		// insert dummy ValidationData
+		let vfp = PersistedValidationData {
+			parent_head: polkadot_parachain::primitives::HeadData(Default::default()),
+			relay_parent_number: 1,
+			relay_parent_storage_root: Default::default(),
+			max_pov_size: 1_000,
+		};
+		<ValidationData<T>>::put(&vfp);
+
+		// insert dummy HostConfiguration with
+		let host_config = AbridgedHostConfiguration {
```

Comment:
Setting all values here to zero may lead to unexpected behavior in the benchmark, if e.g. `set_validation_data` is called within the benchmark.

Reply:
ok, thank you, changed to non-zero values